MailCore/Utils/Constants.swift

```diff
@@ -66,6 +66,7 @@ public struct URLConstants {
     public static let kdriveAppStore = URLConstants(urlString: "https://itunes.apple.com/app/id1482778676")
     public static let importMails = URLConstants(urlString: "https://import-email.infomaniak.com")
     public static let matomo = URLConstants(urlString: "https://analytics.infomaniak.com/matomo.php")
+    public static let sourceCode = URLConstants(urlString: "https://github.com/Infomaniak/ios-kMail")
```

Comment:
I think the name `sourceCode` is not very explicit. What do you think about something like `githubRepository`?